mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -0,0 +1,662 @@
+                mlrun.common.schemas.model_monitoring.SchedulingKeys.APPLICATION_NAME
+            ] = application_name
+        return application_filter_dict
+
+    def delete_model_endpoints_resources(self, endpoints: list[dict[str, typing.Any]]):
+        """
+        Delete all model endpoints resources in both SQL and the time series DB.
+
+        :param endpoints: A list of model endpoints flattened dictionaries.
+        """
+
+        for endpoint_dict in endpoints:
+            endpoint_id = endpoint_dict[
+                mlrun.common.schemas.model_monitoring.EventFieldType.UID
+            ]
+
+            # Delete last analyzed records
+            self.delete_last_analyzed(endpoint_id=endpoint_id)
+
+            # Delete application results records
+            self.delete_application_result(endpoint_id=endpoint_id)
+
+            # Delete model endpoint record
+            self.delete_model_endpoint(endpoint_id=endpoint_id)
+
+    def get_endpoint_real_time_metrics(
+        self,
+        endpoint_id: str,
+        metrics: list[str],
+        start: str = "now-1h",
+        end: str = "now",
+        access_key: str = None,
+    ) -> dict[str, list[tuple[str, float]]]:
+        """
+        Getting metrics from the time series DB. There are pre-defined metrics for model endpoints such as
+        `predictions_per_second` and `latency_avg_5m` but also custom metrics defined by the user.
+
+        :param endpoint_id:      The unique id of the model endpoint.
+        :param metrics:          A list of real-time metrics to return for the model endpoint.
+        :param start:            The start time of the metrics. Can be represented by a string containing an RFC 3339
+                                 time, a Unix timestamp in milliseconds, a relative time (`'now'` or
+                                 `'now-[0-9]+[mhd]'`, where `m` = minutes, `h` = hours, and `'d'` = days), or 0 for the
+                                 earliest time.
+        :param end:              The end time of the metrics. Can be represented by a string containing an RFC 3339
+                                 time, a Unix timestamp in milliseconds, a relative time (`'now'` or
+                                 `'now-[0-9]+[mhd]'`, where `m` = minutes, `h` = hours, and `'d'` = days), or 0 for the
+                                 earliest time.
+        :param access_key:       V3IO access key that will be used for generating Frames client object. If not
+                                 provided, the access key will be retrieved from the environment variables.
+
+        :return: A dictionary of metrics in which the key is a metric name and the value is a list of tuples that
+                 includes timestamps and the values.
+        """
+        # # TODO : Implement this method once Perometheus is supported
+        logger.warning(
+            "Real time metrics service using Prometheus will be implemented in 1.4.0"
+        )
```

Comment:
Probably an outdated message 🤔

Reply:
yes anyway we are going to address it in 1.7.0 so I leave it for now